docs/api.md

```diff
@@ -1,65 +1,134 @@
 ---
 id: api
-title: API
-sidebar_label: API
+title: JavaScript API
+sidebar_label: JavaScript API
 ---
- 
-If a user visits your page and has Web Monetization enabled, your site can
-detect that Web Monetization is enabled.
 
-You can access Web Monetization-related events and state through a Javascript
-API on `document.monetization`.
+This JavaScript browser API allows Web-monetized websites to request small
+payments from users. Payments are sent to the website by the user's [Web
+Monetization provider](/#providers).
+
+> No files are needed to use this API.
+
+This API is injected by any browser or extension that supports Web Monetization.
+
+> This document represents the current API implementation. Note that an
+[unofficial draft specification](../specification.html) suitable for the W3C
+standards track is being developed and should **NOT** be used as a reference for
+current implementations.
+
+#### Assumptions
+
+* Your website is [monetized](./getting-started).
+* The user visiting your site has an account with a Web Monetization provider.
+* The user's browser supports Web Monetization natively or through an extension.
+
+## Document.Monetization
+
+The browser exposes the `document.monetization` DOM object that implements
+[`EventTarget`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget)
+and has a read-only `state` property. The object allows you to track Web
+Monetization events and see whether the user visiting your page is Web monetized.
 
-## Javascript API
 ```ts
 document.monetization: EventTarget
-document.monetization.state: string
+document.monetization.state: 'stopped' | 'pending' | 'started'
 ```
 
-This Javascript API is injected by any browser or extension that supports Web
-Monetization. 
-
-> You don't need to include any files to use this Javascript API.
+## States
 
-Check the value of `document.monetization.state` to see if the user visiting
-your page is Web Monetized.
+Check the value of `document.monetization.state` to see if a user is Web
+Monetized.
 
-There are 3 possible states for Web Monetization.
+### `undefined`
 
-### No Web Monetization
+Until Web Monetization is built into browsers, the `state` will be `undefined`
+unless a polyfill is installed (e.g. a Web Monetization provider's browser
+extension).
 
 ```ts
 document.monetization === undefined
 ```
 
-The user does not support Web Monetization, or there is no **Web Monetization Meta Tag** on the page.
+### `stopped`
 
-### Pending
+The extension is capable of Web Monetization but is not currently sending
+micropayments, nor trying to.
+
+```ts
+document.monetization && document.monetization.state === 'stopped'
+```
+
+If your site is adding a `meta` tag dynamically, then the `state` begins in
```

Comment:
seems I was wrong, in the current extension version the page always starts in the stopped state and then emits a pending event and goes into pending roughly ~100ms after the window.load event is fired. But I think that the desired behavior is for it to immediately start in `pending` when there is a meta tag present, so I think we can keep the docs like this

Reply:
This issue has been fixed and the state will be set to pending immediately upon detecting a meta tag (once the document is ready) and will be published as extension version 42 at some point

Reply:
https://github.com/coilhq/web-monetization-projects/pull/69